src/main/resources/import.sql

```diff
@@ -120,49 +46,49 @@ INSERT INTO startup_platform.investment_interests (id, category_id, sum, descrip
 --   ----------------------------------------------------------------------------------------------
 --  Fill projects table
 --   ----------------------------------------------------------------------------------------------
-INSERT INTO startup_platform.projects (id, name, owner_id, category_id, cost, date_added, date_start, min_invest, description, goal, stage)
+INSERT INTO startup_platform.projects (id, name, owner_id, category_id, cost, date_added, date_start, min_invest, description, goal, STAGE)
 VALUES
-  (1, 'Raccoon.World - gadgets for hands', 1, 1, 400000, '20.05.2017 14:30', '20.05.2017 15:30', 50000, 'Raccoon.World is a company that creates gadgets for hands, based on spatialization, tracking, reproduction of fine motor skills and tactile perception.
+  (1, 'Raccoon.World - gadgets for hands', 1, 1, 400000, '2017-05-20 14:30:30.069', '2017-05-20 15:30:30.069', 50000, 'Raccoon.World is a company that creates gadgets for hands, based on spatialization, tracking, reproduction of fine motor skills and tactile perception.
 Raccoon.world is an international company incorporated in Delaware, USA with R&D Center in Ukraine.', 'goal', 'start'),
-  (2, 'Integrating Virtuality into Reality, Game', 1, 1, 150000, '20.05.2017 14:30', '20.05.2017 15:30', 50000,
+  (2, 'Integrating Virtuality into Reality, Game', 1, 1, 150000, '2017-05-20 14:30:30.069', '2017-05-20 15:30:30.069', 50000,
       'Mobile MMO RPG (midcore), with Step-by-step combat system, and focus on multiplayer real-time PvP.', 'goal',
       'start'),
-  (3, 'Wood Pellets Storage', 1, 2, '1800000', '20.05.2017 14:30', '20.05.2017 15:30', 60000, 'European winter storage
+  (3, 'Wood Pellets Storage', 1, 2, '1800000', '2017-05-20 14:30:30.069', '2017-05-20 15:30:30.069', 60000, 'European winter storage
 All math and figures are taken from personal experience obtained during last year
 Pellets sales market- Europe', 'goal', 'start'),
-  (4, 'Virtual Insurance Agents (chatbots) for insurers', 1, 2, 400000, '20.05.2017 14:30', '20.05.2017 15:30', 20000,
+  (4, 'Virtual Insurance Agents (chatbots) for insurers', 1, 2, 400000, '2017-05-20 14:30:30.069', '2017-05-20 15:30:30.069', 20000,
       'We are building Virtual Insurance Agents (chatbots) to help insurers to engage their customers via Facebook Messengers.',
       'goal', 'middle'),
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal 'middle' 4 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.com/coding_rules#rule_key=plsql%3AS1192)